### ISSUE TITLE
Discuss WAF with IAS

### ISSUE THREAD

**Issue by marisastrong** [CDLUC3/mrt-doc#1004]
WAF - Would like to just try it
- See what would be blocked before we actually enable it?
e.g. Turn on in stage; but real trial will be prod
- Concerns over possible blocking of valid transactions being block, such as Dryad
- Keep track of the effects of WAF
Put rule in place w/counter to observe actions and number of actions that would potentially occur; sounds ideal in terms of evaluation

For any new services in the future, use of a WAF could be a good option by default

**Comment by marisastrong** [comment]
Met with IAS - next steps
- [ ] read the IAS Users Guide on WAF
- [ ] Invite Martin/Joe to the next meeting to discuss how to implement

Discussed usage logs that would be helpful in understand what is being blocked by rules in addition to count

**Comment by terrywbrady** [comment]
Categories
- wordpress
- php
- linux system file names
- bad login requests (422)
- paths that do not conform to standard patterns
  - Slightly out of date, but many captured here: https://github.com/terrywbrady/mrt-log-analyzer/blob/master/ui.py

**Comment by terrywbrady** [comment]
CDLSYS request sent on 4/19.

**Comment by marisastrong** [comment]
sent CDLSYS ticket for running WAF commands on the UC3 Ops box

**Comment by terrywbrady** [comment]
Implementation in progress... tracking issues in a new ticket
